config/load_test.go

```diff
@@ -587,6 +587,22 @@ func TestLoad(t *testing.T) {
 			},
 		},
 		{
+			args: []string{"-ui.certfile", "cert_value"},
+			cfg: func(cfg *Config) *Config {
+				cfg.UI.CertFile = "cert_value"
+				cfg.Registry.Consul.CheckScheme = "https"
+				return cfg
+			},
+		},
+		{
+			args: []string{"-ui.keyfile", "key_value"},
+			cfg: func(cfg *Config) *Config {
+				cfg.UI.KeyFile = "key_value"
+				cfg.Registry.Consul.CheckScheme = "http"
```

Comment:
No it won't. Question is whether this should then be an error and fabio refuse to start since this config doesn't make sense.